src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -742,6 +740,17 @@ function copyOptions() {
         $(this).prop('checked', $prevTable.eq(index).prop('checked'));
     });
 
+    // Hide visibility options and update common visibility options dropdown text if a common option is selected
+    var prevQuestionVisibilityOption = $prevQuestionForm.find('.visibility-options-dropdown > button').html();
+    $newQuestionForm.find('.visibility-options-dropdown > button').html(prevQuestionVisibilityOption);
```

Comment:
Maybe use `.text()` for this line and the one above instead of `.html()`, although both have the same effect here it seems  like a more appropriate method to use